userbenchmark/distributed/ddp_experiments.py

```diff
@@ -0,0 +1,282 @@
+import argparse
```

Comment:
Since this experiment is owned by compiler team, not distributed, I suggest to move this to `userbenchmark/ddp_experiment/__init__.py`